LayoutKitTests/OverlayLayoutTests.swift

```diff
@@ -111,10 +185,42 @@ class OverlayLayoutTests: XCTestCase {
                 1.0: expectedFrame,
                 2.0: expectedFrame,
                 3.0: expectedFrame
-            ])
+                ])
         }
     }
 
+    /**
+     Tests primary layouts smaller than overlay.
+     Sublayouts should have same measurements.
+     */
+    func testPrimaryLayoutsSmallerThanOverlay() {
+        let primaryLayout0 = SizeLayout<View>(width: 2, height: 2)
+        let primaryLayout1 = SizeLayout<View>(width: 3, height: 4)
+        let sublayout = SizeLayout<View>(width: 10, height: 10)
+
+        let overlay = OverlayLayout(primaryLayouts: [primaryLayout0, primaryLayout1], overlayLayouts: [sublayout])
+        let arrangement = overlay.arrangement(width: 3.0, height: 4.0)
```

Comment:
Why is it important to limit the available space here to match the largest primary layout? It seems like there's two mechanisms that could be limiting the resulting frames in this test, the size of the primary layouts and the size of the available space. It might be worthwhile splitting this up into separate tests to test these individually.

Reply:
Fixed by splitting it into two tests.